pkg/server/testserver_http.go

```diff
@@ -88,17 +88,19 @@ func (ts *httpTestServer) GetUnauthenticatedHTTPClient() (http.Client, error) {
 
 // GetAdminHTTPClient implements the TestServerInterface.
 func (ts *httpTestServer) GetAdminHTTPClient() (http.Client, error) {
-	httpClient, _, err := ts.getAuthenticatedHTTPClientAndCookie(authenticatedUserName(), true)
+	httpClient, _, err := ts.getAuthenticatedHTTPClientAndCookie(authenticatedUserName(), true, false)
 	return httpClient, err
 }
 
 // GetAuthenticatedHTTPClient implements the TestServerInterface.
-func (ts *httpTestServer) GetAuthenticatedHTTPClient(isAdmin bool) (http.Client, error) {
+func (ts *httpTestServer) GetAuthenticatedHTTPClient(
+	isAdmin bool, multiTenantSession bool,
+) (http.Client, error) {
```

Comment:
It's test-code, so I don't feel strongly, but if you wanted, the callsites might be a bit more clear with something like (warning, code written in browser window):

```golang
type SessionType int

const (
    SingleTenantSession SessionType = 0
    MultiTenantSession  SessionType = 1
)

func (ts *httpTestServer) GetAuthenticatedHTTPClient(
	isAdmin bool, sessionType SessionType,
) (http.Client, error) {
```